mattermost-plugin/plugin.json

```diff
@@ -7,7 +7,7 @@
     "release_notes_url": "https://github.com/mattermost/focalboard/releases",
     "icon_path": "assets/starter-template-icon.svg",
     "version": "0.9.0",
-    "min_server_version": "5.37.0",
+    "min_server_version": "5.38.0",
```

Comment:
We cannot depend yet on `"6.0.0"` as the version does not exist in `mattermost-server`

Reply:
Yup, that's dependent on mattermost-server [PR-18247](https://github.com/mattermost/mattermost-server/pull/18247). We should update our dependency to v6.0 once that change is rolled out to community and community-daily, which should happen on Aug 25 ([thread](https://community.mattermost.com/core/pl/hrq9g5m6y7d75gftaxyp7iiu3y)).